From 2586696eeef666b31654e0cbf1b586ceb1687889 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Mon, 13 Nov 2017 09:25:04 +0800
Subject: [PATCH] clk: rockchip: export the i2s sclk parent clocks for rk3036

Add the SCLK_I2S_PRE and SCLK_I2S_FRAC id for i2s sclk.

Change-Id: Ic66291f2a76ec5c2a47e9721ad6f37922264fd42
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 include/dt-bindings/clock/rk3036-cru.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/dt-bindings/clock/rk3036-cru.h b/include/dt-bindings/clock/rk3036-cru.h
index 8224da212da3..764d58644a5e 100644
--- a/include/dt-bindings/clock/rk3036-cru.h
+++ b/include/dt-bindings/clock/rk3036-cru.h
@@ -52,6 +52,8 @@
 #define SCLK_PVTM_CORE		123
 #define SCLK_PVTM_GPU		124
 #define SCLK_PVTM_VIDEO		125
+#define SCLK_I2S_FRAC		126
+#define SCLK_I2S_PRE		127
 #define SCLK_MAC		151
 #define SCLK_MACREF		152
 #define SCLK_MACPLL		153
-- 
2.35.3

